src/main.rs

```diff
@@ -80,6 +80,9 @@ pub struct InstallOpts {
     /// Relative or full path for the export file that will be generated. If no path is provided, the file will be generated under home directory (https://docs.rs/dirs/latest/dirs/fn.home_dir.html).
     #[arg(short = 'f', long)]
     pub export_file: Option<PathBuf>,
+    /// The path to which the espup configuration file will be written to.
+    #[arg(long = "config", value_parser = XtensaRust::parse_version)]
+    pub espup_config_path: Option<PathBuf>,
```

Comment:
```suggestion
    pub config_path: Option<PathBuf>,
```
I would suggest removing `espup` from the argument name